src/psyclone/domain/lfric/lfric_arg_descriptor.py

```diff
@@ -442,7 +442,7 @@ def _init_field(self, arg_type, operates_on):
                            const.VALID_ANY_SPACE_NAMES)
 
         # Check accesses for kernels that operate on DoFs
-        if operates_on == "dof":
+        if operates_on in ["dof", "owned_dof"]:
```

Comment:
Maybe we are done adding OPERATES_ON, but here and on line 545, is there a LFRicConstant that represents this set of values? If not, should there be?

Reply:
Oh yes. I've updated them.